Sources/InkCLI/Printing.swift

```diff
@@ -0,0 +1,44 @@
+/**
+*  Ink
+*  Copyright (c) John Sundell 2019
+*  MIT license, see LICENSE file for details
+*/
+
+import Foundation
+
+internal var versionMessage: String = "Ink (v0.1.2): Markdown -> HTML converter"
```

Comment:
Unless we automate this, there's a very little chance that it'll stay up to date, since it'd require manually modifying the source code each time I make a new release. I suggest dropping it, at least for now.

Reply:
Yeah, I thought of that. Not a big deal to omit.